neo4j-cli/aura/internal/subcommands/deployment/deployment.go

```diff
@@ -0,0 +1,58 @@
+package deployment
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/neo4j/cli/common/clicfg"
+	"github.com/neo4j/cli/common/clierr"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/subcommands/deployment/database"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/subcommands/deployment/server"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/subcommands/deployment/token"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmd(cfg *clicfg.Config) *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "deployment",
+		Short: "Relates to Fleet Manager deployments",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			cfg.Aura.BindBaseUrl(cmd.Flags().Lookup("base-url"))
+
+			cfg.Aura.BindAuthUrl(cmd.Flags().Lookup("auth-url"))
+
+			outputValue := cmd.Flags().Lookup("output").Value.String()
+			if outputValue != "" {
+				validOutputValue := false
```

Comment:
```suggestion
			for _, v := range clicfg.ValidOutputValues {
					if v == outputValue {
						cfg.Aura.BindOutput(cmd.Flags().Lookup("output"))
						
						return nil
					}
				}
				
				return clierr.NewUsageError("invalid output value specified: %s", outputValue)
```
Instead of introducing a valid flag, would this be cleaner? What do you think